src/main/java/io/appium/java_client/Location.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client;
+
+/**
+ * Represents the physical location.
+ */
+public class Location extends org.openqa.selenium.html5.Location {
+    private final double latitude;
+    private final double longitude;
+    private final double altitude;
+
+    /**
+     * Create {@link Location} with latitude, longitude and altitude values.
+     *
+     * @param latitude latitude value.
+     * @param longitude longitude value.
+     * @param altitude altitude value.
+     */
+    public Location(double latitude, double longitude, double altitude) {
```

Comment:
altitude is not required and might be null

Reply:
Selenium require primitive here, if we break inheritance, we'll need to introduce some ugly method in `SupportsLocation`: e.g. `default io.appium.java_client.Location appiumLocation()`

Reply:
Having only latitude and longitude better reflects how actual APIs work (e.g. for in xcuitest altitude is not used/supported, uia2 and espresso allow to set location without the altitude)